.github/scripts/.build.zsh

```diff
@@ -188,21 +189,21 @@ build() {
       }
       popd
       ;;
-    linux-*)
+    ubuntu-*)
       local cmake_bin='/usr/bin/cmake'
       cmake_args+=(
-        -S ${PWD} -B build_${target##*-}
-        -G Ninja
-        -DCMAKE_BUILD_TYPE:STRING=${config}
-        -DCEF_ROOT_DIR:PATH="${project_root}/.deps/cef_binary_${CEF_VERSION}_${target//-/_}"
-        -DENABLE_AJA:BOOL=OFF
-        -DENABLE_WEBRTC:BOOL=OFF
+        --preset ubuntu-ci
+        --toolchain ${project_root}/cmake/linux/toolchain-${target##*-}-gcc.cmake
+        -DENABLE_BROWSER:BOOL=ON
         -DENABLE_NATIVE_NVENC:BOOL=OFF
+        -DCEF_ROOT_DIR:PATH="${project_root}/.deps/cef_binary_${CEF_VERSION}_${target//ubuntu-/linux_}"
       )
+
       if (( ! UBUNTU_2210_OR_LATER )) cmake_args+=(-DENABLE_NEW_MPEGTS_OUTPUT:BOOL=OFF)
+      if [[ ${target##*-} == aarch64 ]] cmake-args+=(-DENABLE_QSV11:BOOL=OFF)
 
-      cmake_build_args+=(build_${target##*-} --config ${config} --parallel)
-      cmake_install_args+=(build_${target##*-} --prefix ${project_root}/build_${target##*-}/install/${config})
+      cmake_build_args+=(build_${target%%-*} --config ${config} --parallel)
+      cmake_install_args+=(build_${target%%-*} --prefix ${project_root}/build_${target%%-*}/install/${config})
```

Comment:
Correct. Windows is the only platform with an arch-specific build directory because it needs to have a 32-bit and 64-bit build configuration side-by-side.

As the scripts are CI-only, no such accommodation needs to be made for macOS or Ubuntu builds as they will always target a single architecture and no parallel build directories are needed.

Reply:
Got it.